### PR TITLE
Remove typeof typos

### PR DESCRIPTION
**Pull request by morganpizzini**
Remove double typeof

### CODE REVIEW

**Comment by mtziortzis** [comment]
This is causing some issues indeed. Thanks @morganpizzini  for fixing this.
When is this scheduled to be merged ?

**Comment by matteotumiati** [comment]
@jahaganiev do you have time to review and merge this one, and perhaps release 2.5.1? This is quite a blocker for the ones running 2.5.0 given we're unable to load our webapps. Thanks!

**Comment by jahaganiev** [comment]
@olegpix can please have a look into it?

**Comment by jahaganiev** [comment]
Just to updated that this PR will be merged with the upcoming v2.5.1 release soon. Thanks!

File: src/index.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ if (typeof _ !== 'undefined' && typeof Dropzone !== 'undefined')
 else HSFileUploadModule = null;
 export { HSFileUploadModule as HSFileUpload };
 
-if (typeof typeof noUiSlider !== 'undefined')
+if (typeof noUiSlider !== 'undefined')
 	HSRangeSliderModule = require('./plugins/range-slider').default;
```
Comment (mohsinhijazee) [inline src/index.ts:51]: Noob question - I am using it for an internal tool and came across this. Why are we using `require` here? Especially when require is not available in browsers and locally linking this package (`pnpm link /local/clone/of/preline`) throws an error that `require` is not defined which it isn't in the browsers if I understand correctly. 

I replaced it with following:

```typescript
	(async () => {
		HSRangeSliderModule = (await import('./plugins/range-slider')).default;
	})();
```

would that be wrong?

**Pull request merged**